Services/Init/classes/class.ilInitialisation.php

```diff
@@ -311,7 +311,7 @@ protected static function initClientIniFile()
 			}
 			else
 			{
-				self::abortAndDie("Invalid client");
+				self::abortAndDie("Fatal Error: ilInitialisation::initClientIniFile initializing client ini file abborted with: ". $ilClientIniFile->ERROR);
```

Comment:
Yes i did. E.g if the does not exist ne message will be like this.

`Fatal Error: ilInitialisation::initClientIniFile initializing client ini file abborted with: file_does_not_exist`